test/data.yml

```diff
@@ -11,32 +11,17 @@ users:
     last_name: Aardvark
   - first_name: Amanda
     last_name: Aardvark
-queries:
-  - id: "#user_query"
-    model: thelook
-    view: users
+queries_system_activity:
```

Comment:
@jkaster These data.yml file changes are fine with me, ok for you?

Reply:
for sure, if `yarn test` works for the Typescript tests from the root of the codegen repo

Reply:
@muckweaver on second thought, can we revert this top level yaml key back to `queries`? leave the body the same but update the pytest fixture back to `queries` etc. that will make it so we don't have to update all the other SDK tests